$NetBSD: patch-skins_larry_mail.css,v 1.1 2013/06/15 16:08:09 taca Exp $

Fix from repository:
* commit 63049afbfe457d9b1dd709e7d8b108218c5ea3f2:
  Set minimum height for message headers block (#1489108)

--- skins/larry/mail.css.orig	2013-05-16 18:06:37.000000000 +0000
+++ skins/larry/mail.css
@@ -151,7 +151,7 @@ a.iconbutton.threadmode.selected {
 	white-space: nowrap;
 	overflow: hidden;
 	text-overflow: ellipsis;
-	background-image: url(images/listicons.png?v=bd98.25486);
+	background-image: url(images/listicons.png);
 	background-repeat: no-repeat;
 	background-position: 6px 3px;
 }
@@ -254,7 +254,7 @@ a.iconbutton.threadmode.selected {
 	left: 19px;
 	width: 13px;
 	height: 13px;
-	background: url(images/listicons.png?v=bd98.25486) -3px -144px no-repeat;
+	background: url(images/listicons.png) -3px -144px no-repeat;
 	cursor: pointer;
 }
 
@@ -376,7 +376,7 @@ a.iconbutton.threadmode.selected {
 	right: 4px;
 	width: 20px;
 	height: 18px;
-	background: url(images/buttons.png?v=abf1.36693) -3px -458px no-repeat;
+	background: url(images/buttons.png) -3px -458px no-repeat;
 }
 
 #mailpreviewtoggle.enabled {
@@ -463,7 +463,7 @@ a.iconbutton.threadmode.selected {
 #messagelist thead tr td.sortedDESC a {
 	color: #004458;
 	text-decoration: underline;
-	background: url(images/listicons.png?v=bd98.25486) right -912px no-repeat;
+	background: url(images/listicons.png) right -912px no-repeat;
 }
 
 #messagelist thead tr td.sortedASC a {
@@ -526,7 +526,7 @@ a.iconbutton.threadmode.selected {
 	height: 18px;
 	width: 20px;
 	padding: 0;
-	background: url(images/listicons.png?v=bd98.25486) -100px 0 no-repeat;
+	background: url(images/listicons.png) -100px 0 no-repeat;
 }
 
 #messagelist tbody tr td.attachment span.attachment {
@@ -826,7 +826,7 @@ h3.subject {
 	left: 0;
 	width: 18px;
 	height: 16px;
-	background: url(images/buttons.png?v=abf1.36693) -27px -242px no-repeat;
+	background: url(images/buttons.png) -27px -242px no-repeat;
 }
 
 .moreheaderstoggle.remove .iconlink {
@@ -846,7 +846,7 @@ div.more-headers {
 	width: 12px;
 	height: 10px;
 	cursor: pointer;
-	background: url(images/buttons.png?v=abf1.36693) center -1579px no-repeat;
+	background: url(images/buttons.png) center -1579px no-repeat;
 }
 
 div.hide-headers {
@@ -903,7 +903,7 @@ div.hide-headers {
 	width: 32px;
 	height: 32px;
 	overflow: hidden;
-	background: url(images/contactpic_32px.png?v=09a3.287) center center no-repeat #fff;
+	background: url(images/contactpic_32px.png) center center no-repeat #fff;
 	border-radius: 3px;
 }
 
@@ -913,6 +913,10 @@ div.hide-headers {
 	border-radius: 3px;
 }
 
+#messageheader .message-headers {
+	min-height: 60px;
+}
+
 #messageheader #contactphoto {
 	display: block;
 	position: absolute;
@@ -923,7 +927,7 @@ div.hide-headers {
 	overflow: hidden;
 	border-radius: 4px;
 	border: 1px solid #e6e6e6;
-	background: url(images/contactpic_48px.png?v=1bc4.353) center center no-repeat #fff;
+	background: url(images/contactpic_48px.png) center center no-repeat #fff;
 }
 
 #messageheader #contactphoto img {
@@ -1200,7 +1204,7 @@ div.message-partheaders .headers-table t
 }
 
 #compose-contacts li a, #contacts-table td {
-	background: url(images/listicons.png?v=bd98.25486) -100px 0 no-repeat;
+	background: url(images/listicons.png) -100px 0 no-repeat;
 	overflow: hidden;
 	padding-left: 36px;
 	text-overflow: ellipsis;
@@ -1412,7 +1416,7 @@ div.message-partheaders .headers-table t
 }
 
 #compose-attachments.droptarget {
-	background-image: url(images/filedrop.png?v=deab.605);
+	background-image: url(images/filedrop.png);
 	background-position: center bottom;
 	background-repeat: no-repeat;
 }
